src/lv_objx/lv_label.h

```diff
@@ -66,7 +66,7 @@ typedef struct
     union{
         char * dot_tmp_ptr; /* Pointer to the allocated memory containing the character which are replaced by dots (Handled by the library)*/
         char dot_tmp[ sizeof(char *) ];    /* Directly store the characters if <=4 characters */
-    };
+    } dt;
```

Comment:
It seems clearer to me:
```c
union{
        char * tmp_ptr; /* Pointer to the allocated memory containing the character which are replaced by dots (Handled by the library)*/
        char tmp[ sizeof(char *) ];    /* Directly store the characters if <=4 characters */
    } dot;
```

What do you think?

Reply:
I agree. I renamed it to that.